test/functional/utils-specs.js

```diff
@@ -0,0 +1,87 @@
+import { fs, mkdirp } from 'appium-support';
+import nodefs from 'fs';
+import { copyGradleProjectRecursively } from '../../lib/utils';
+import os from 'os';
+import path from 'path';
+import chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+
+chai.should();
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+describe('copyGradleProjectRecursively', function () {
+  let baseSrcDir;
+  let baseDestDir;
+
+  function expectNotExist (dir) {
+    expect(function () {
+      fs.accessSync(dir, fs.constants.F_OK);
+    }).to.throw();
+  }
+
+  async function expectCorrectFileContentIn (filepath) {
+    await fs.readFile(filepath, 'utf8').should.eventually.eql('foobar');
+  }
+
+  async function createTestFile (filepath) {
+    await fs.writeFile(filepath, 'foobar', 'utf8');
+  }
+
+  beforeEach(function () {
+    baseSrcDir = nodefs.mkdtempSync(path.join(os.tmpdir(), 'appium-src-'));
+    baseDestDir = nodefs.mkdtempSync(path.join(os.tmpdir(), 'appium-dst-'));
+  });
+
+  afterEach(async function () {
+    await fs.rimraf(baseSrcDir);
+    await fs.rimraf(baseDestDir);
```

Comment:
My understanding of `rimraf` docs is that this method doesn't fail when directory doesn't exist. Could someone confirm this?

Reply:
yes it does not